pkgs/development/compilers/rust/binary.nix

```diff
@@ -26,7 +26,8 @@ rec {
     inherit src;
 
     meta = with lib; {
```

Comment:
Also why is meta not near the end of the mkDerivation function?

Reply:
no idea.